Ugh I wrote a reply before but did not send, multitasking is killing focus o_O

```
../nuttx/tools/checkpatch.sh -c -u -m -g 773f69d5d95d0390d60447a29c2c1b82931932a3..HEAD
ERROR __main__.py:618: Check failed: /home/runner/work/nuttx-apps/nuttx-apps/apps/graphics/lvgl/CMakeLists.txt

cmake-format check failed, run following command to update the style:
  $ cmake-format <src> -o <dst>

ERROR __main__.py:618: Check failed: /home/runner/work/nuttx-apps/nuttx-apps/apps/graphics/lvgl/CMakeLists.txt
Used config files:
    1: .codespellrc
Some checks failed. For contributing guidelines, see:
  https://github.com/apache/nuttx/blob/master/CONTRIBUTING.md
Error: Process completed with exit code 1.
```

You can check locally with:

> Quick Check for Compliance
> 
> Please verify formatting of your git commits before a Pull Request is submitted/updated. Use [checkpatch.sh](https://nuttx.apache.org/docs/latest/components/tools/checkpatch.html#checkpatch-sh) script that will verify the coding style, encoding, and spelling issues:
> 
>     ./tools/checkpatch.sh -c -u -m -g HEAD~...HEAD
> 
> Alternatevily you can run this script passing the .c file or .h header you want to check using -f switch:
> 
>     ./tools/checkpatch.sh -c -u -m -f path/to/your/file.c

See: https://nuttx.apache.org/docs/latest/contributing/coding_style.html#quick-check-for-compliance

:-)

--- a/graphics/lvgl/CMakeLists.txt
+++ b/graphics/lvgl/CMakeLists.txt
@@ -27,12 +27,21 @@ if(CONFIG_GRAPHICS_LVGL)
   # ############################################################################
   set(LVGL_DIR ${CMAKE_CURRENT_LIST_DIR}/lvgl)
 
+  set(LVGL_VERSION
+      "${CONFIG_LVGL_VERSION_MAJOR}.${CONFIG_LVGL_VERSION_MINOR}.${CONFIG_LVGL_VERSION_PATCH}"
+  )
+
   if(NOT EXISTS ${LVGL_DIR})
+    set(LVGL_URL
+        "https://github.com/lvgl/lvgl/archive/refs/tags/v${LVGL_VERSION}.zip")
     FetchContent_Declare(
       lvgl_fetch
-      DOWNLOAD_DIR ${CMAKE_CURRENT_LIST_DIR}
-      URL "https://github.com/lvgl/lvgl/archive/refs/tags/v9.2.1.zip" SOURCE_DIR
-          ${CMAKE_CURRENT_LIST_DIR}/lvgl BINARY_DIR
+      URL ${LVGL_URL}
+          DOWNLOAD_DIR
+          ${CMAKE_CURRENT_LIST_DIR}
+          SOURCE_DIR
+          ${CMAKE_CURRENT_LIST_DIR}/lvgl
+          BINARY_DIR
           ${CMAKE_BINARY_DIR}/apps/graphics/lvgl/lvgl
       DOWNLOAD_NO_PROGRESS true
       TIMEOUT 120)

--- a/graphics/lvgl/Kconfig
+++ b/graphics/lvgl/Kconfig
@@ -11,7 +11,7 @@ menuconfig GRAPHICS_LVGL
 
 if GRAPHICS_LVGL
 
-# Kconfig file for LVGL v9.2.1
+# Kconfig file for LVGL v9.2.2
 
 menu "LVGL configuration"
 
@@ -1495,7 +1495,7 @@ menu "LVGL configuration"
 			default 2 # LVGL_VERSION_MINOR
 		config LVGL_VERSION_PATCH
 			int
-			default 1 # LVGL_VERSION_PATCH
+			default 2 # LVGL_VERSION_PATCH
 	endmenu
 
 	menu "Devices"

--- a/graphics/lvgl/Makefile
+++ b/graphics/lvgl/Makefile
@@ -58,8 +58,11 @@ WD := ${shell echo $(CURDIR) | sed -e 's/ /\\ /g'}
 
 CONFIG_GRAPH_LVGL_URL ?= "https://github.com/lvgl/lvgl/archive/refs/tags"
 
-LVGL_VERSION = 9.2.1
-LVGL_TARBALL = v$(LVGL_VERSION).zip
+_MAJ         := $(CONFIG_LVGL_VERSION_MAJOR)
+_MIN         := $(CONFIG_LVGL_VERSION_MINOR)
+_PAT         := $(CONFIG_LVGL_VERSION_PATCH)
+LVGL_VERSION := $(_MAJ).$(_MIN).$(_PAT)
+LVGL_TARBALL := v$(LVGL_VERSION).zip
 
 LVGL_UNPACKNAME = lvgl
 UNPACK ?= unzip -o $(if $(V),,-q)

